.github/workflows/test.yml

```diff
@@ -1,32 +1,32 @@
-name: Test
-
-on:
-  pull_request:
-    branches:
-      - "master"
-
-jobs:
-  test:
-    runs-on: ubuntu-latest
-    timeout-minutes: 10
-
-    steps:
-      - name: Checkout repo
-        uses: actions/checkout@v2
-
-      - name: Set Up Python 3.10
-        uses: actions/setup-python@v2
-        with:
-          python-version: "3.10"
-
-      - name: Install requirements
-        run: |
-          python -m pip install --upgrade pip
-          pip install -r requirements.txt
-
-      - name: Run flake8
-        run: flake8
-
-      - name: Run tests
-        timeout-minutes: 5
-        run: python manage.py test
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - "master"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Set Up Python 3.10
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.10"
+
+      - name: Install requirements
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+
+      - name: Run flake8
+        run: flake8
+
+      - name: Run tests
+        timeout-minutes: 5

```

Comment:
The `timeout-minutes` setting here might be redundant since the job-level timeout is already set to 10 minutes. Consider removing this line unless you specifically need a shorter timeout for this step.